src/account/operations/transaction/options.rs

```diff
@@ -59,8 +61,7 @@ pub struct TransactionOptionsDto {
     pub custom_inputs: Option<Vec<OutputId>>,
     #[serde(rename = "mandatoryInputs", default)]
     pub mandatory_inputs: Option<Vec<OutputId>>,
-    #[serde(rename = "allowBurning", default)]
-    pub allow_burning: bool,
+    pub burn: Option<BurnDto>,
```

Comment:
Bindings need to be adapted for this change (and it should be in the changelog)

Reply:
😡😡😡